Kitodo-LongTermPreservationValidation/src/main/java/org/kitodo/longtermpreservationvalidation/jhove/KitodoJhoveBase.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.longtermpreservationvalidation.jhove;
+
+import edu.harvard.hul.ois.jhove.App;
+import edu.harvard.hul.ois.jhove.JhoveBase;
+import edu.harvard.hul.ois.jhove.JhoveException;
+import edu.harvard.hul.ois.jhove.Module;
+import edu.harvard.hul.ois.jhove.RepInfo;
+
+import java.io.File;
+import java.lang.reflect.UndeclaredThrowableException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.api.validation.longtermpreservation.FileType;
+import org.kitodo.api.validation.longtermpreservation.LtpValidationConditionInterface;
+import org.kitodo.api.validation.longtermpreservation.LtpValidationConditionResult;
+import org.kitodo.api.validation.longtermpreservation.LtpValidationError;
+import org.kitodo.api.validation.longtermpreservation.LtpValidationResult;
+import org.kitodo.api.validation.longtermpreservation.LtpValidationResultState;
+import org.kitodo.longtermpreservationvalidation.conditions.LtpValidationConditionEvaluator;
+
+/**
+ * Manages Jhove.
+ * 
+ * <p>The png module (PNG-gdm, "com.mcgath.jhove.module.PngModule") is disabled, because it 
+ * depends on the package "jhove-ext-modules". For some reason, including this jar interfers 
+ * with the xml processing in Kitodo.Production somehow. Specifically, mods2kitodo schema 
+ * conversion does not work correctly.</p>
```

Comment:
Comment: It sounds like the module is changing the JVM-wide default XML parser. Yes, I had something like that myself with a third web application running parallel to Kitodo, and it was disrupting Kitodo. We have to explicitly call the XML parser constructor everywhere in Kitodo, instead of using the default constructor from the interface. We usually do this, but maybe not always. It's also bad (if I'm right) if the module does that; it would affect the entire Tomcat system.

Nothing needs to be changed here.